### DISCUSSION TITLE
Welcome to pyGenomeViz Discussions!

### DISCUSSION THREAD

**Discussion by moshi4** [moshi4/pyGenomeViz#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by chrissikath** [comment]
Hey,

I really love your package, Thanks a lot!
 I wonder how i can customize the content inside the hover element. 
 For example i want to dont give a track a label but show the label additionally with the location and length (what is default):
 
 Greetings!
